ui/client/http/HttpService.js

```diff
@@ -87,6 +87,14 @@ export default {
     return promiseWrap($.get(API_URL + '/settings'))
   },
 
+  fetchPlugins() {
```

Comment:
I think better way is using axios to fetch data - now it's avaiable at master.

Reply:
updated